### PR TITLE
Issue 585 - Add support for autoPaymentOff

### PR DESCRIPTION
**Pull request by javier-gomez-hs**

### CODE REVIEW

**Comment by pierre** [comment]
:shipit:

File: profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestEntitlement.java [hunk 1]
```diff
@@ -299,6 +300,36 @@ public void testCreateEntitlementInTheFuture() throws Exception {
...
+        final Account accountJson = createAccount();
+        assertNotNull(accountJson);
+
+        // assign autoPaymentOff tag to account
+        Tags tags = killBillClient.createAccountTag(accountJson.getAccountId(), new UUID(0L, 1L), requestOptions);
+        assertEquals(tags.get(0).getTagDefinitionName(), "AUTO_PAY_OFF");
+
+        // verify that number of invoices and payments for account is still 0
+        assertEquals(killBillClient.getInvoicesForAccount(accountJson.getAccountId(), requestOptions).size(), 0);
+        assertEquals(killBillClient.getPaymentsForAccount(accountJson.getAccountId(), requestOptions).size(), 0);
```
Comment (sbrossie) [inline profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestEntitlement.java]: Confused.. Why would setting `AUTO_PAY_OFF` prevent the system from creating invoices?

Comment (sbrossie) [inline profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestEntitlement.java]: Scratch that comment... got confused with your comment. This is unrelated to the tag. All good.

**Comment by sbrossie** [comment]
👍
